packages/client/snowpack.config.js

```diff
@@ -0,0 +1,31 @@
+const environment = process.env.NODE_ENV || 'development'
+const development = environment === 'development'
+
+/** @type {import("snowpack").SnowpackUserConfig } */
+module.exports = {
+  mount: {
+    public: '/',
+    src: '/_dist_',
+  },
+  plugins: [
+    [
+      '@snowpack/plugin-run-script',
+      {
+        name: 'Server Console',
+        cmd: development ? 'cd ../server && yarn start' : 'true',
+      },
+    ],
+    '@snowpack/plugin-dotenv',
+    '@snowpack/plugin-typescript',
+    '@prefresh/snowpack',
+  ],
+  installOptions: {
+    installTypes: true,
+  },
+  proxy: {
+    '/graphql': {
+      target: 'http://localhost:3000',
+      changeOrigin: true,
+    },
```

Comment:
This is neat!

Reply:
Yeah it is! This is one of those features where the moment I saw the docs I asked myself why it's not a standard option in every frontend toolchain ever.

The best thing about it is that this way we can completely ignore CORS (since as far as both sides of the proxy are concerned we never cross any origins).